websites/W/Wolfy/presence.ts

```diff
@@ -1,3 +1,106 @@
+	consultingSkin: string;
+	consultingSpecificLeaderboard: string; // $leaderboardType
+	joinGame: string;
+	laurels: string;
+	life: string;
+	lookingAtProfile: string; // $username
+	lunar: string;
+	inAMenu: string;
+	participatingToAnEvent: string;
+	playing: string;
+	readingAnArticle: string;
+	toConsultArticle: string;
+	toConsultCategory: string;
+	toConsultEvent: string;
+	visitHelpCenter: string;
+	visitProfile: string; // $username
+	visitWolfy: string;
+	waiting: string;
+};
+
+const strings: Record<string, Partial<langStrings>> = {
+	en: {
+		choosingSkin: "Choosing a skin",
+		consultingArticles: "Consulting articles",
+		consultingCategory: "Consulting a category ⤵️",
+		consultingAFutureEvent: "Consulting a future event ⤵️",
+		consultingEvents: "Consulting events",
+		consultingHelpCenter: "Consulting the help center",
+		consultingHome: "Consulting the home page",
+		consultingLeaderboard: "Consulting the leaderboard",
+		consultingShop: "Consulting the shop",
+		consultingSettings: "Consulting the settings",
+		consultingSkin: "Consulting a skin",
+		consultingSpecificLeaderboard:
+			"Consulting the $leaderboardType leaderboard",
+		joinGame: "Join game",
+		laurels: "Laurels",
+		life: "Eternal",
+		lookingAtProfile: "Looking at $username's profile",
+		lunar: "Lunar",
+		inAMenu: "In a menu",
+		participatingToAnEvent: "Participating to an event",
+		playing: "In Game",
+		readingAnArticle: "Reading an article ⤵️",
+		toConsultArticle: "Consult this article",
+		toConsultCategory: "Consult this category",
+		toConsultEvent: "Consult this event",
+		visitHelpCenter: "Visit the help center",
+		visitProfile: "Visit $username's profile",
+		visitWolfy: "Visit Wolfy's website",
+		waiting: "WAITING"
+	},
+	fr: {
+		choosingSkin: "Choisit son skin",
+		consultingArticles: "Consulte les articles",
+		consultingCategory: "Consulte la catégorie ⤵️",
+		consultingAFutureEvent: "Consulte l'événement prochain ⤵️",
+		consultingEvents: "Consulte les évènements",
+		consultingHelpCenter: "Consulte le centre d'aide",
+		consultingHome: "Consulte la page d'accueil",
+		consultingLeaderboard: "Consulte le classement",
+		consultingShop: "Consulte la boutique",
+		consultingSettings: "Consulte ses paramètres",
+		consultingSkin: "Consulte un skin",
+		consultingSpecificLeaderboard: "Consulte le classement $leaderboardType",
+		joinGame: "Rejoindre la partie",
+		laurels: "Lauriers",
+		life: "Éternel",
+		lookingAtProfile: "Regarde le profil de $username",
+		lunar: "Lunaire",
+		inAMenu: "Dans un menu",
+		participatingToAnEvent: "Participe à un événement",
+		playing: "En jeu",
+		readingAnArticle: "Lit l'article ⤵️",
+		toConsultArticle: "Consulter l'article",
+		toConsultCategory: "Consulter la catégorie",
+		toConsultEvent: "Consulter l'événement",
+		visitHelpCenter: "Visite le centre d'aide",
+		visitProfile: "Voir le profil de $username",
+		visitWolfy: "Visiter le site de Wolfy",
+		waiting: "EN ATTENTE"
+	}
+};
```

Comment:
We do not allow multiLang to be added in this way
> Presences should use [common details](https://api.premid.app/v2/langFile/presence/en) (strings starting with "general."). You can achieve this using multiLanguage with the provided strings. If your presence requires custom strings, then you shouldn't use multiLanguage until the presence gets 1000 users. You can find an example [here](https://docs.premid.app/dev/presence/class#getstringsobject).

Reply:
well that's pretty sad, it was suppose to come the release of Wolfy english games.

I guess i will just make the fix and event handle then :)